.github/workflows/e2e.yml

```diff
@@ -22,14 +22,14 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v4
+      - uses: actions/setup-go@v5
         with:
           go-version: "1.22"
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.7.0
+        uses: golangci/golangci-lint-action@v6.1.0
         with:
-          version: v1.59
+          version: v1.60
```

Comment:
I changed to use 1.60 because there is a memory bug in 1.59 that causes a lot of problems on my machine.


The version bumps in this file is just because I thought it was good to have latest version 🤷